Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -24,13 +24,80 @@
 #if ENABLE_SWIFTUI
 
 import Foundation
+import Observation
+import SwiftUI // FIXME: Do not import SwiftUI in WebKit proper.
```

Comment:
Do we have a task about a cross import overlay?

Something you might want to front load is opting in to the new "internal imports by default" mode, which would also require you to qualify these as `public import` instead. That should prevent accidentally exposing frameworks going forward, and will be easier to do from the get go.

https://www.swift.org/blog/using-upcoming-feature-flags/
https://github.com/swiftlang/swift-evolution/blob/main/proposals/0409-access-level-on-imports.md

Reply:
I will update the PR to reflect the bug tracking the overlay adoption. I will also update the PR to qualify the access level imports. I will enable that default import level feature in a subsequent PR